SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -111,6 +111,8 @@ class FLocalDeploymentManager final
 
 	FString ExposedRuntimeIP;
 
+	FString LastSnapshotPath;
```

Comment:
```suggestion
	FString CurrentSnapshotDirectory;
```
`LastSnapshotPath` reads to me like the path to the latest snapshot that was taken.

Reply:
Sure I'm fine with that.